Sources/Models/PublicEpisodes/0010-ATaleOfTwoFlatMaps.swift

```diff
@@ -27,33 +27,130 @@ operation to other structures and derive new, useful code!
 )
 
 private let exercises: [Episode.Exercise] = [
-  Episode.Exercise(problem:
-"""
+  Episode.Exercise(
+    problem: """
 Define `filtered` as a function from `[A?]` to `[A]`.
-"""),
-  Episode.Exercise(problem:
-    """
+""",
+    solution: """
+```
+func filtered<A>() -> ([A?]) -> [A] {
```

Comment:
I think we want the signature to be `([A?]) -> [A]` rather than `() -> ([A?]) -> [A]`, but I'm also unsure of if we wanna mix functions and methods. @mbrandonw do you have a preference on using methods everywhere vs. functions in these exercises?

Reply:
@dannyhertz yoooo did you wanna make these changes?? if you flatten these extra `() -> ` functions in a few of your solutions, I think it might be g2g!